marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mapping/getDocumentForTesting.sjs

```diff
@@ -56,5 +59,6 @@ if (rtn.format.toUpperCase() === 'JSON') {
   rtn.data = transformResult.data;
   rtn.namespaces = transformResult.namespaces;
 }
+rtn.sourceProperties = sourceProps.buildSourceProperties(rtn.data, isJson);
```

Comment:
One other small tweak - I am not wild about abbreviations. :) For consistency too, rename "rtn" to "response"; we have a number of other DS endpoints that return something named "response".

Reply:
Addressed in https://github.com/marklogic/marklogic-data-hub/pull/4871.